typescript/sdk/src/metadata/warpRouteConfig.ts

```diff
@@ -10,7 +10,7 @@ const TokenConfigSchema = z.object({
   type: z.nativeEnum(TokenType),
   hypAddress: z.string(), // HypERC20Collateral, HypERC20Synthetic, HypNativeToken address
   tokenAddress: z.string().optional(), // external token address needed for collateral type eg tokenAddress.balanceOf(hypAddress)
-  tokenCoinGeckoId: z.string().optional(), // CoinGecko id for token
+  tokenCoingeckoId: z.string().optional(), // CoinGecko id for token
```

Comment:
dumb nit but any reason not to keep it as `tokenCoinGeckoId`? We do `CoinGecko` for the registry's `gasCurrencyCoinGeckoId`, consistency probably makes sense

Reply:
Ah fair, will make the change!